website/docs/en/plugins/dev/core.mdx

```diff
@@ -298,6 +300,8 @@ type TransformHandler = (
 The `handler` function provides the following params:
 
 - `code`: The code of the module.
+- `context`: The directory path of the currently processed module.
```

Comment:
We can link to https://rspack.dev/api/loader-api/context#thiscontext because it has more details